applications/nrf5340_audio/src/modules/audio_sync_timer.h

```diff
@@ -17,6 +17,13 @@
  */
 uint32_t audio_sync_timer_capture(void);
 
+/**
+ * @brief Add some documentation :)
+ * 
+ * @retval
+*/
+uint32_t audio_sync_frame_send_timer_capture(void);
```

Comment:
Never used. Can we omit these two files?

Reply:
For testing purposes, I didn't have to use it. For the real thing, I think it might have to be used to make the solution more robust, due to how timestamped SDUs works on our controller. I think we should discuss the scenario where I think this is needed elsewhere.